server/routes/reviewPlan/reviewCheckYourAnswersController.ts

```diff
@@ -1,11 +1,17 @@
 import { RequestHandler } from 'express'
 import ReviewCheckYourAnswersView from './reviewCheckYourAnswersView'
+import { getPrisonerContext } from '../../data/session/prisonerContexts'
 
 export default class ReviewCheckYourAnswersController {
   getReviewCheckYourAnswersView: RequestHandler = async (req, res, next): Promise<void> => {
+    const { prisonNumber } = req.params
     const { prisonerSummary } = res.locals
+    const userName = res.locals.user.name
```

Comment:
I've just noticed in the screenshot in the jira ticket, that when the user did the review themselves, then on the Check Your Answers page, we should render "I did the review myself" rather than their name:
![Screenshot 2024-11-01 at 08 51 39](https://github.com/user-attachments/assets/5879a51b-c233-4400-97a3-d88a82a32b1c)

This is a good thing as it means we dont need this code here to get the username, and we dont need it as a property on the `ReviewCheckYourAnswersView` class either 👍

Reply:
Ah, I had been following the ACs for this one. Checked with Karen and the screenshot is correct.

![Selected November 01 2024 14:18:37](https://github.com/user-attachments/assets/8eccefc8-9bb0-48b8-853b-7fdd9a97f2d4)